dotnet/src/Functions/Functions.OpenApi/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -214,18 +216,23 @@ internal static List<RestApiOperation> CreateRestApiOperations(OpenApiDocument d
                     responses: CreateRestApiOperationExpectedResponses(operationItem.Responses).ToDictionary(static item => item.Item1, static item => item.Item2),
                     securityRequirements: CreateRestApiOperationSecurityRequirements(operationItem.Security)
                 )
-                {
-                    Extensions = CreateRestApiOperationExtensions(operationItem.Extensions, logger)
-                };
+                    {
+                        Extensions = CreateRestApiOperationExtensions(operationItem.Extensions, logger)
+                    };
 
-                operations.Add(operation);
+                    operations.Add(operation);
+                }
+                catch (KernelException ke)
+                {
+                    logger.LogWarning(ke, "Error occurred creating REST API operation for {0}. Operation will be ignored.", operationItem.OperationId);
```

Comment:
```suggestion
                    logger.LogWarning(ke, "Error occurred creating REST API operation for {OperationId}. Operation will be ignored.", operationItem.OperationId);
```
Use [log message templating](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#log-message-template) instead of string interpolation.